tests/suites/test_suite_psa_crypto_init.function

```diff
@@ -54,7 +54,6 @@ static int fake_entropy_source( void *state_arg,
 
 #define ENTROPY_SOURCE_PLATFORM                 0x00000001
 #define ENTROPY_SOURCE_TIMING                   0x00000002
-#define ENTROPY_SOURCE_HAVEGE                   0x00000004
 #define ENTROPY_SOURCE_HARDWARE                 0x00000008
 #define ENTROPY_SOURCE_NV_SEED                  0x00000010
```

Comment:
Remove missing gap from macros. It does not appear to have been hardcoded in anywhere else so this should be safe to do.
```suggestion
#define ENTROPY_SOURCE_PLATFORM                 0x00000001
#define ENTROPY_SOURCE_TIMING                   0x00000002
#define ENTROPY_SOURCE_HARDWARE                 0x00000004
#define ENTROPY_SOURCE_NV_SEED                  0x00000008
```